native/packages/react-native-bpk-component-picker/src/BpkPickerMenu.ios.js

```diff
@@ -92,7 +92,11 @@ const BpkPickerMenu = (props: Props) => {
       </TouchableWithoutFeedback>
       <View style={styles.modal}>
         <View style={styles.modalHeader}>
-          <BpkButtonLink title={doneLabel} onPress={onClose} />
+          <BpkButtonLink
+            allowFontScaling={false}
```

Comment:
Yup this is now in the correct place, however we now face the issue that BpkButtonLink doesn't pass through `allowFontScaling` to the underlying `BpkText` 😬